src/components/gateway/WithdrawForm.vue

```diff
@@ -33,9 +33,14 @@
       </div>
       <div>
         <h6>{{ $t('components.gateway.withdraw_form_modal.balance') }} {{ balance | tokenAmount(tokenInfo.decimals)}} {{ token }}</h6>
-        <h6 v-if="isWithdrawalLimitEnabled && isCheckDailyRemainingWithdrawAmount()">
+        <h6 v-if="reachedLimit()">
+          {{ $t('components.gateway.withdraw_form_modal.daily_withdrawal_limit_reached') }}
+          {{ isWithdrawalLimitEnabled | tokenAmount(tokenInfo.decimals) }} {{ token }}
```

Comment:
This doesn't make sense, `isWithdrawalLimitEnabled` returns a boolean, how is that going to render an amount?